### DISCUSSION TITLE
Replit

### DISCUSSION THREAD

**Discussion by PetalCat** [PSDTools/app#18] [General]
Just a reminder that any code on replit is public facing. Do not put your API keys in code use environment variables please god. Anyone can see your code hosted with replit

**Comment by lishaduck** [comment]
If that was aimed at me, yeah, I know. However, isn't the repl private?

**Reply by ParkerH27** [reply]
Repls are not private unless you pay, Also from what I can tell replit does not run flutter so we may have to move to another dev environment such as a vs code live share server

**Reply by lishaduck** [reply]
Yeah. Repl.it seems to have better multi-cursor support, but I don't really care.

**Reply by ParkerH27** [reply]
made an issue for a new dev environment

**Reply by lishaduck** [reply]
Linking it: PSDTools/app#5
